File: 프로그래머스/lv0/120893. 대문자와 소문자/대문자와 소문자.py
def solution(my_string):
    answer = ''
    
    for x in my_string:
        if x.isupper():  # 대문자인 경우
            answer += x.lower()  # 소문자로 변경
        else:
            answer += x.upper()  # 소문자인 경우 대문자로 변경
            
    return answer

File: 프로그래머스/lv0/120893. 대문자와 소문자/대문자와 소문자.js
function solution(my_string) {
    let arr = [...my_string]
    let new_arr = [];  // 새로운 배열 생성
    
    arr.forEach((x) => {
        // toUpperCase 대문자로 변환
        // toLowerCase 소문자로 변환
        if (x === x.toUpperCase()) {
            new_arr.push(x.toLowerCase())
        }
        else {
            new_arr.push(x.toUpperCase())
        }
    })
    
    return new_arr.join('')
    
}